src/control/cmd/ddb/ddb_commands.go

```diff
@@ -0,0 +1,301 @@
+			f.Bool("r", "recursive", false, "Recursively list the contents of the path")
+
+		},
+		Args: func(a *grumble.Args) {
+			a.String("path", "Optional, list contents of the provided path", grumble.Default(""))
+		},
+		Run: func(c *grumble.Context) error {
+			return ls_wrapper(ctx, c.Args.String("path"), c.Flags.Bool("recursive"))
+		},
+		Completer: nil,
+
+	})
+	app.AddCommand(&grumble.Command {
+		Name:      "open",
+		Aliases:   nil,
+		Help:      "Opens the vos file at <path>",
+		LongHelp:  `Opens the vos file at <path>. The '-w' option allows for modifying the vos file
+with the rm, load, commit_ilog, etc commands. The path <path> should be an absolute path to the
+pool shard. Part of the path is used to determine what the pool uuid is. `,
+		HelpGroup: "",
+		Flags: func(f *grumble.Flags) {
+			f.Bool("w", "write_mode", false,
+				"Open the vos file in write mode.")
+
+		},
+		Args: func(a *grumble.Args) {
+			a.String("path", "Path to the vos file to open. ")
+		},
+		Run: func(c *grumble.Context) error {
+			return open_wrapper(ctx, c.Args.String("path"), c.Flags.Bool("write_mode"))
+		},
+		Completer: openCompleter,
+	})
+	app.AddCommand(&grumble.Command {
+		Name:      "close",
+		Aliases:   nil,
+		Help:      "Close the currently opened vos pool shard",
+		LongHelp:  "",
+		HelpGroup: "",
+		Run: func(c *grumble.Context) error {
+			return close_wrapper(ctx)
+		},
+		Completer: nil,
+	})
+	app.AddCommand(&grumble.Command {
+		Name:      "dump_superblock",
+		Aliases:   nil,
+		Help:      "Dump the pool superblock information",
+		LongHelp:  "",
+		HelpGroup: "",
+		Run: func(c *grumble.Context) error {
+			return dump_superblock_wrapper(ctx)
+		},
+		Completer: nil,
+	})
+	app.AddCommand(&grumble.Command {
+		Name:      "dump_value",
+		Aliases:   nil,
+		Help:      "Dump a value to a file",
+		LongHelp:  `Dump a value to a file. The vos path should be a complete
+path, including the akey and if the value is an array value it should include
+the extent.`,
+		HelpGroup: "",
+		Args: func(a *grumble.Args) {
+			a.String("path", "VOS tree path to dump.")
+			a.String("dst", "File path to dump the value to.")
+		},
+		Run: func(c *grumble.Context) error {
+			return dump_value_wrapper(ctx, c.Args.String("path"), c.Args.String("dst"))
+		},
+		Completer: nil,
+	})
+	app.AddCommand(&grumble.Command {
+		Name:      "rm",
+		Aliases:   nil,
+		Help:      "Remove a branch of the VOS tree.",
+		LongHelp:  `Remove a branch of the VOS tree. The branch can be anything from a container
+and everything under it, to a single value.`,
+		HelpGroup: "",
+		Args: func(a *grumble.Args) {
+			a.String("path", "VOS tree path to remove.")
+		},
+		Run: func(c *grumble.Context) error {
+			return rm_wrapper(ctx, c.Args.String("path"))
+		},
+		Completer: nil,
+	})
+	app.AddCommand(&grumble.Command {
+		Name:      "load",
+		Aliases:   nil,
+		Help:      "Load a value to a vos path.",
+		LongHelp:  `Load a value to a vos path. This can be used to update
```

Comment:
```suggestion
	app.AddCommand(&grumble.Command{
		Name:    "load",
		Aliases: nil,
		Help:    "Load a value to a vos path.",
		LongHelp: `Load a value to a vos path. This can be used to update
```